Description: Build-depend on KNotifications for KNotificationJobUiDelegate 
 src/kded/daemon.cpp:11 and and src/controller.cpp:20 are using
 `KNotificationJobUiDelegate` so `plasma-welcome` should build depend on
 `KNotifications`.
 .
 This would also match what is already set in [.kde-ci.yml](.kde-ci.yml#L14).
Author: Aurélien COUDERC <coucouf@debian.org>
Forwarded: https://invent.kde.org/plasma/plasma-welcome/-/merge_requests/188
Last-Update: 2025-01-18
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -51,7 +51,7 @@
 )
 
 find_package(Qt6 ${QT_MIN_VERSION} REQUIRED COMPONENTS Core Gui Network Qml QuickControls2 Svg)
-find_package(KF6 ${KF6_MIN_VERSION} REQUIRED COMPONENTS Kirigami Config CoreAddons ConfigWidgets DBusAddons I18n KIO NewStuff Service WindowSystem DBusAddons KCMUtils Svg JobWidgets)
+find_package(KF6 ${KF6_MIN_VERSION} REQUIRED COMPONENTS Kirigami Config CoreAddons ConfigWidgets DBusAddons I18n KIO NewStuff Notifications Service WindowSystem DBusAddons KCMUtils Svg JobWidgets)
 find_package(Plasma ${PROJECT_DEP_VERSION} REQUIRED)
 find_package(KF6KirigamiAddons 1.2.0 REQUIRED)
 
